tests/VolleyManagement.UnitTests/Services/GameReportService/Comparers/StandingsEntryComparer.cs

```diff
@@ -12,6 +12,27 @@
     [ExcludeFromCodeCoverage]
     internal class StandingsEntryComparer : IComparer<StandingsEntry>, IComparer
     {
+        private bool arePointsComparer;
```

Comment:
Would you be able to tell me what this name means in 3 months time?

Reply:
maybe areComparerByPoints?

Reply:
shouldComaprerByPoints?

Reply:
Sofiia is closer